docker-compose.local.yml

```diff
@@ -3,21 +3,54 @@ version: '3'
 services:
   db:
     image: postgres
+    restart: always
+    healthcheck:
+      interval: 10s
+      retries: 10
+      test:
+        [
+          'CMD-SHELL',
+          'pg_isready'
+        ]
+      timeout: 45s
     environment:
       POSTGRES_DB: bc
       POSTGRES_USER: bc
       POSTGRES_PASSWORD: bc
+      PGUSER: bc
+      PGPASSWORD: bc
```

Comment:
You already set username and password for the superuser. In such case, you need to change the healthcheck


```
test:
  [
    'CMD',
    'pg_isready',
    '-U',
    '${POSTGRES_USER}',
    '-P',
    '${POSTGRES_PASSWORD}',
  ]
```